linera-explorer/graphql/operations_schema.graphql

```diff
@@ -25,53 +20,40 @@ A Sha3-256 value
 scalar CryptoHash
 
 
-type HashedChainOperation {
-	operation: ChainOperation!
-	hash: CryptoHash!
-}
-
 
 
```

Comment:
Four blank lines.

Reply:
it's generated by the `async-graphql` library